float64 takeoff_height
PiecewiseTrajectory desired_trajectory
float64 land_height
---
---
uint8 TAKING_OFF = 0
uint8 FOLLOWING = 1
uint8 LANDING = 2
uint8 DEADLOCKED = 3
uint8 CRASHED = 4
uint8 current_state